modules/profile/files/jenkins/agent_gpu/etc/lightdm/xhost.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+xhost +si:localuser:jenkins-agent
+touch /tmp/xhost_`date +"%T"`
```

Comment:
The discussion here was lost in the UI when I moved these files. ([Original link](https://github.com/ros-infrastructure/buildfarm_deployment/pull/211#discussion_r265741569))

> debug code to help with the problem of knowing when the good configuration is loaded into the X server

Should this be removed before the PR merges or do we want to keep this debug code in production? Should we append the date to a file rather than touching a file in /tmp to avoid inode limits on long running instances?

Reply:
we can remove the touch command without any problem.